pkg/crd/client.go

```diff
@@ -60,6 +65,20 @@ func (cg *ClientGenerator) getRestConfig() (*rest.Config, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	qps, _ := utils.GetUIntValueFromEnv(EnvKubeClientQps)
```

Comment:
Should we use `utils.GetIntValueFromEnv` instead of `utils.GetUIntValueFromEnv`?

Reply:
Or rather use `utils.GetUIntValueFromEnv` only to get max QPS and max ClientBurst since these are always positive integers.

Reply:
actual usage is in float and int so sticked to types as suggested by linter and code scanner

```
cg.restConfig.QPS = float32(qps)
cg.restConfig.Burst = burst
```